### ISSUE TITLE
Command buffer overflow in hf tune?

### ISSUE THREAD

**Issue by lesbeben** [Proxmark/proxmark3#81]
Hi everyone,

I was playing around with hf tune, trying a bunch of RFID cards to sse which were HF, when I encountered some kind of bug.

I usually just run hf tune and check for a drop in voltage when the card is close to my antenna. Today, however, after a little while (around a minute, maybe less) the client started displaying:

WARNING: Command buffer about to overwrite command! This needs to be fixed!

instead of the usual "#db# XXXXXmV". Just thought I'd let you know, since I'm in no way proficient enough to fix this myself (yet).

I'm running a one-week old client (git pull today on master, but did not compile yet) with firmware 2.0.0-rc2.

Thanks as usual!

TL;DR: Ran "hf tune" for around a minute, got "WARNING: Command buffer about to overwrite command! This needs to be fixed!" :kissing_heart:

**Comment by holiman** [comment]
I think this is ok, IIRC, this basically says that commands coming in from the device faster than we're handling them. In 'normal' operation, this would be bad, thus the printout that it needs to be fixed. In this particular case, however, where the device just loops spitting out information about the voltage, it's not an issue. 

On the other hand, perhaps we should limit the voltage printouts a bit.

**Comment by iceman1001** [comment]
yeah, there is alot of printouts with the "hf tune" command.  maybe slow it down to twice per second?
